tests/pipelines/test_pipelines_audio_classification.py

```diff
@@ -66,6 +68,10 @@ def run_pipeline_test(self, audio_classifier, examples):
 
         self.run_torchaudio(audio_classifier)
 
+        spec_output_keys = {field.name for field in fields(AudioClassificationOutputElement)}
+        model_output_keys = set(output[0].keys())
```

Comment:
Correct me if I'm wrong, but shouldn't we iterate on all the outputs rather than just 0?

Reply:
Fixed!